pkg/services/ngalert/tests/util.go

```diff
@@ -22,6 +22,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func init() {
+	rand.Seed(42)
```

Comment:
```suggestion
	rand.Seed(time.Now().UnixNano())
```

Reply:
Random seed is great but without knowing what it was when a test failed it can be hard to reproduce the failure.